pkg/proxy/apis/kubeproxyconfig/validation/validation.go

```diff
@@ -140,17 +141,24 @@ func validateKubeProxyConntrackConfiguration(config kubeproxyconfig.KubeProxyCon
 	return allErrs
 }
 
-func validateProxyMode(mode kubeproxyconfig.ProxyMode, fldPath *field.Path) field.ErrorList {
+func validateProxyMode(config *kubeproxyconfig.KubeProxyConfiguration, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	switch mode {
+
+	ipvsFeatureGate := fmt.Sprintf("%s=true", features.SupportIPVSProxyMode)
+	if config.Mode == kubeproxyconfig.ProxyModeIPVS && !strings.Contains(config.FeatureGates, ipvsFeatureGate) {
```

Comment:
Please don't use `!strings.Contains(config.FeatureGates, ipvsFeatureGate)`

You may need `!utilfeature.DefaultFeatureGate.Enabled(SupportIPVSProxyMode)`?

Reply:
Good point! Updated.